internal/exec/stages/files/units.go

```diff
@@ -186,9 +186,6 @@ func (s *stage) writeSystemdUnit(unit types.Unit, runtime bool) error {
 	return s.Logger.LogOp(func() error {
 		relabeledDropinDir := false
 		for _, dropin := range unit.Dropins {
-			if dropin.Contents == nil || *dropin.Contents == "" {
```

Comment:
Would it make sense to write the unit if `""` but not if `nil`?

Reply:
I think it would make more sense to me from a user perspective that it would write it regardless of if I put an empty string or not including the field at all but I'm not hard against doing it this way.